examples/basic_functionality/avoid_openai_rate_limits.py

```diff
@@ -0,0 +1,39 @@
+from time import sleep, time
+
+# When you're creating OpenAI embeddings, you will run up against the rate limit.  In this example, the rate limit is 10,000 requests per minut.  
+# If the collection is too big, we will run against the 10,000 requests per minute limit, and get a rate limit error.
+# Call collection.add with a slice of 1000 of the data at each time.  Then sleep for 45 seconds.
+
+def add_embeddings(tech_data_list_description, tech_data_list_uuid, tech_data_meta):
+  '''
+  tech_data_list_description - The list of data to be added that will be vectorized.
+  tech_data_list_uuid - The id's.  In our case will be the uuids.
+  tech_data_meta - The metadata.  In our case will be the source of the data.
+  '''
+
+  
+  while len(tech_data_list_description) > 1000:
+      time_start = time()
+      print(f"Start Loop: {len(tech_data_list_description)}")
+      collection.add(
+          documents=tech_data_list_description[:1000],       # The list of data to be added that will be vectorized.
+          ids=tech_data_list_uuid[:1000],                    # The id's.  In our case will be the uuids.
+          metadatas=tech_data_meta[:1000]             # The metadata.  In our case will be the source of the data.
+      )
+      tech_data_list_description = tech_data_list_description[1000:]
+      tech_data_list_uuid = tech_data_list_uuid[1000:]
+      tech_data_meta = tech_data_meta[1000:]
+      print(f"Technologies To Go: {len(tech_data_list_description)}")
```

Comment:
I think this will always print `len(tech_data_list_description) - 1000` since we don't actually shorten the list anywhere. If we update the above log line to print the indices we're currently working on then I think we can remove this log line, what do you think?